bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/robot/StatusDTO.java

```diff
@@ -85,6 +85,12 @@ public class StatusDTO {
     @SerializedName("mop_forbidden_enable")
     @Expose
     private Integer mopForbiddenEnable;
+    @SerializedName("dry_status")
+    @Expose
+    private Integer isMapDryingActive;
```

Comment:
Typo?
```suggestion
    private Integer isMopDryingActive;
```

Reply:
In deed: Typo: I'm going to fix that, too.

Thanks